<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="SupplierSuite" duration-ms="638623" started-at="2018-01-09T14:38:47Z" finished-at="2018-01-09T14:49:26Z">
    <groups>
    </groups>
    <test name="SupplierTestone" duration-ms="144152" started-at="2018-01-09T14:38:47Z" finished-at="2018-01-09T14:41:11Z">
      <class name="Booking_MyAdmin.Book_Prepay_MultiSupplier">
        <test-method status="PASS" signature="BookPrepayMultiSupplier(java.lang.String)[pri:0, instance:Booking_MyAdmin.Book_Prepay_MultiSupplier@8e24743]" name="BookPrepayMultiSupplier" duration-ms="141610" started-at="2018-01-09T20:08:47Z" finished-at="2018-01-09T20:11:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[CH]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BookPrepayMultiSupplier -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:Booking_MyAdmin.Book_Prepay_MultiSupplier@8e24743]" name="afterTest" is-config="true" duration-ms="2526" started-at="2018-01-09T20:11:09Z" finished-at="2018-01-09T20:11:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:Booking_MyAdmin.Book_Prepay_MultiSupplier@8e24743]" name="getResult" is-config="true" duration-ms="0" started-at="2018-01-09T20:11:09Z" finished-at="2018-01-09T20:11:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=BookPrepayMultiSupplier status=SUCCESS method=Book_Prepay_MultiSupplier.BookPrepayMultiSupplier(java.lang.String)[pri:0, instance:Booking_MyAdmin.Book_Prepay_MultiSupplier@8e24743] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
      </class> <!-- Booking_MyAdmin.Book_Prepay_MultiSupplier -->
    </test> <!-- SupplierTestone -->
    <test name="SupplierTesttwo" duration-ms="162420" started-at="2018-01-09T14:41:11Z" finished-at="2018-01-09T14:43:54Z">
      <class name="Booking_MyAdmin.Book_Credit_MultiSupplier">
        <test-method status="FAIL" signature="BookCreditMultiSupplier(java.lang.String)[pri:0, instance:Booking_MyAdmin.Book_Credit_MultiSupplier@43a0cee9]" name="BookCreditMultiSupplier" duration-ms="155442" started-at="2018-01-09T20:11:11Z" finished-at="2018-01-09T20:13:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[CH]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Timed out after 30 seconds waiting for visibility of element located by By.xpath: //div[contains(text(),'Confirmed')]
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'PC350015', ip: '10.245.98.86', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.30.477700 (0057494ad8732195794a7b32078424f92a5fce41), userDataDir=C:\Users\479739\AppData\Local\Temp\scoped_dir25216_4848}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=58.0.3029.110, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: e018819d5862239c6ddc6619735151f3 expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Timed out after 30 seconds waiting for visibility of element located by By.xpath: //div[contains(text(),'Confirmed')]
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'PC350015', ip: '10.245.98.86', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.30.477700 (0057494ad8732195794a7b32078424f92a5fce41), userDataDir=C:\Users\479739\AppData\Local\Temp\scoped_dir25216_4848}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=58.0.3029.110, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: e018819d5862239c6ddc6619735151f3 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at Booking_MyAdmin.Book_Credit_MultiSupplier.BookCreditMultiSupplier(Book_Credit_MultiSupplier.java:406)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BookCreditMultiSupplier -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:Booking_MyAdmin.Book_Credit_MultiSupplier@43a0cee9]" name="getResult" is-config="true" duration-ms="153" started-at="2018-01-09T20:13:47Z" finished-at="2018-01-09T20:13:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=BookCreditMultiSupplier status=FAILURE method=Book_Credit_MultiSupplier.BookCreditMultiSupplier(java.lang.String)[pri:0, instance:Booking_MyAdmin.Book_Credit_MultiSupplier@43a0cee9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:Booking_MyAdmin.Book_Credit_MultiSupplier@43a0cee9]" name="afterTest" is-config="true" duration-ms="6618" started-at="2018-01-09T20:13:47Z" finished-at="2018-01-09T20:13:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- Booking_MyAdmin.Book_Credit_MultiSupplier -->
    </test> <!-- SupplierTesttwo -->
    <test name="SupplierTestthree" duration-ms="331958" started-at="2018-01-09T14:43:54Z" finished-at="2018-01-09T14:49:26Z">
      <class name="Booking_MyAdmin.Book_Credit_Card_MultiSupplier">
        <test-method status="FAIL" signature="BookCreditCardMultiSupplier(java.lang.String)[pri:0, instance:Booking_MyAdmin.Book_Credit_Card_MultiSupplier@47db50c5]" name="BookCreditCardMultiSupplier" duration-ms="331904" started-at="2018-01-09T20:13:54Z" finished-at="2018-01-09T20:19:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[CH]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Sheet index (2) is out of range (0..0) expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Sheet index (2) is out of range (0..0) expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at Booking_MyAdmin.Book_Credit_Card_MultiSupplier.BookCreditCardMultiSupplier(Book_Credit_Card_MultiSupplier.java:426)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BookCreditCardMultiSupplier -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:Booking_MyAdmin.Book_Credit_Card_MultiSupplier@47db50c5]" name="afterTest" is-config="true" duration-ms="32" started-at="2018-01-09T20:19:26Z" finished-at="2018-01-09T20:19:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:Booking_MyAdmin.Book_Credit_Card_MultiSupplier@47db50c5]" name="getResult" is-config="true" duration-ms="0" started-at="2018-01-09T20:19:26Z" finished-at="2018-01-09T20:19:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=BookCreditCardMultiSupplier status=FAILURE method=Book_Credit_Card_MultiSupplier.BookCreditCardMultiSupplier(java.lang.String)[pri:0, instance:Booking_MyAdmin.Book_Credit_Card_MultiSupplier@47db50c5] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
      </class> <!-- Booking_MyAdmin.Book_Credit_Card_MultiSupplier -->
    </test> <!-- SupplierTestthree -->
  </suite> <!-- SupplierSuite -->
</testng-results>
